opal/browser/audio/param_schedule.rb

```diff
@@ -0,0 +1,24 @@
+module Browser; module Audio
+
+  class ParamSchedule
+    include Native
+
+    alias_native :value,               :setValueAtTime
+    alias_native :linear_ramp_to,      :linearRampToValueAtTime
+    alias_native :exponential_ramp_to, :exponentialRampToValueAtTime
+    alias_native :cancel_all,          :cancelScheduledValues
+
+    def initialize(audio_param)
+      super Native.convert(audio_param)
+    end
+
+    def target(target, time_hash)
+      `#@native.setTargetAtTime(target, time_hash[:start_time], time_hash[:time_constant])`
```

Comment:
This isn't gonna work. You have to interpolate the `time_hash` accesses or it will be a syntax error.